doc/source/cluster/cluster_under_construction/key-concepts.rst

```diff
@@ -1,130 +1,78 @@
-.. warning::
-    This page is under construction!
+.. include:: /_includes/clusters/we_are_hiring.rst
 
 Key Concepts
 ============
-..
-    TODO(cade) Can we simplify this? From https://github.com/ray-project/ray/pull/26754#issuecomment-1192927645:
-    * Worker Nodes
-    * Head Node
-    * Autoscaler
-    * Clients and Jobs
-    
-    Need to add the following sections + break out existing content into them.
-    See ray-core/user-guide.rst for a TOC example
-    
-    overview
-    high-level-architecture
-    jobs
-    nodes-vs-workers
-    scheduling-and-autoscaling
-    configuration
-    Things-to-know
-
-.. include:: /_includes/clusters/we_are_hiring.rst
 
 .. _cluster-key-concepts-under-construction:
 
-Key Concepts
-============
-
-Cluster
--------
 
-A Ray cluster is a set of one or more nodes that are running Ray and share the
-same :ref:`head node<cluster-node-types>`.
+This page introduces the following key concepts, an understanding of which is helpful when creating and managing Ray clusters:
 
-.. _cluster-node-types-under-construction:
+.. contents::
+    :local:
 
-Node types
-----------
+.. _cluster-worker-nodes-under-construction:
 
-A Ray cluster consists of a :ref:`head node<cluster-head-node>` and a set of
-:ref:`worker nodes<cluster-worker-node>`.
+Worker nodes
+------------
+A Ray cluster consists of a set of one or more **worker nodes**. Ray tasks and actors
+can be scheduled onto the worker nodes. Each worker node runs helper processes which
+implement distributed scheduling and :ref:`memory management<memory>`.
 
-.. image:: ray-cluster.jpg
+.. figure:: ray-cluster.jpg
     :align: center
     :width: 600px
 
-.. _cluster-head-node-under-construction:
-
-Head node
-~~~~~~~~~
+    A Ray cluster consists of a set of one or more worker nodes, one of which is designated
```

Comment:
I'd remove the word "worker" in this sentence.
We usually categorize workers and head into disjoint buckets.

Reply:
I've intentionally removed the notion that they're disjoint buckets. The distinction necessitates explanation on why they're different (e.g. `Node Types`), whereas here a head node is simply a worker node plus additional things.

Thoughts?

This isn't original btw, I was inspired by the [simple explanations in the whitepaper](https://docs.google.com/document/d/1lAy0Owi-vPz2jEqBSaHNQcy2IBSDEHyXNOQZlGuj93c/preview#heading=h.y4i8cfntvi1k) :)

Reply:
Oh I guess that is how the whitepaper explains it.

Well, they're different in that a head node has additional things.
Often the head node requires different configuration than worker nodes. (This is the case in our sample configs.)
For large workloads we often recommend hacking the head node with num-cpus=0 to avoid scheduling Ray workloads there and make it more-or-less into a control node.

If the head node is a special kind of worker node, I guess "worker" is redundant -- we just have nodes, one of which is a head. Eliminating the terminology "worker node" altogether would also reduce the conflation between "worker nodes" and "worker processes", but I think this ship has sailed.

Anyways, I think I'll defer to the authorities on this one :)

Reply:
In an ideal state, there would be no head node -- just worker nodes, and some control processes which can run wherever.

Reply:
+1 to @DmitriGekhtman, let's remove the term "worker node".

Reply:
My 2c: 
- it seems that it's better to keep the "cluster" as a key concept. We could emphasize that ray cluster just include one or more nodes and explain briefly the distinction between head node and worker node. After this brief explanation, we could use "head node" and "worker node" in the rest of the doc.
- we should revamp the diagram to highlight the differences between the head node and worker node precisely and concisely.
- The note "Here, a node means an individual machine. However, [on Kubernetes](https://ray--27510.org.readthedocs.build/en/27510/cluster/cluster_under_construction/ray-clusters-on-kubernetes/index.html#kuberay-index), nodes are implemented as pods." could also be moved under Ray cluster.

------- here is how it might look like ------

### Ray cluster
A Ray cluster consists of one or more nodes, one of which is designated the head node while the rest are often called worker nodes.
<img width="833" alt="Screen Shot 2022-08-05 at 1 33 48 PM" src="https://user-images.githubusercontent.com/9677264/183158561-d29a82aa-56cd-42c2-8a03-f687f86d30a2.png">

The number of worker nodes in a cluster may change with application demand, according to your Ray cluster configuration. The [head node](https://ray--27510.org.readthedocs.build/en/27510/cluster/cluster_under_construction/key-concepts.html#cluster-head-node-under-construction) runs the logic which implements autoscaling.

<img width="775" alt="Screen Shot 2022-08-05 at 1 39 51 PM" src="https://user-images.githubusercontent.com/9677264/183159300-897f070d-4f6e-4cdd-8230-05639a0a281c.png">

Reply:
Thanks for the feedback, everyone! This is implemented.